cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -55,7 +56,11 @@ message CreateSeismicRequest {
     string external_id = 2;  // not optional
     Identifier partition = 3;
     string seismic_store_id = 4;
-    VolumeDef volume_def = 5;  // Defines the portion of the tracestore volume to limit the new seismic to 
+    // Defines the portion of the tracestore volume to limit the new seismic to 
```

Comment:
I'm thinking maybe making the Volume extent its own message might give us a little more flexibility down the road... But maybe I'm overthinking it. I'm probably overthinking it.